k8s/keycloak/schema.yaml

```diff
@@ -0,0 +1,126 @@
+x-google-marketplace:
+  schemaVersion: v2
+
+  applicationApiVersion: v1beta1
+
+  publishedVersion: "$TAG"
+  publishedVersionMetadata:
+    releaseNote: >-
+      Initial release.
+    releaseTypes:
+      - Feature
+    recommended: false
+
+  clusterConstraints:
+    resources:
+    - replicas: 1
+      requests:
+        cpu: 100m
+        memory: 100Mi
+
+  images:
+    '':
+      properties:
+        keycloak.image.repo:
+          type: REPO_WITH_REGISTRY
+        keycloak.image.tag:
+          type: TAG
+    postgresql:
+      properties:
+        postgresql.image.repo:
+          type: REPO_WITH_REGISTRY
+        postgresql.image.tag:
+          type: TAG
+
+  deployerServiceAccount:
+    description:
+      Creates app resources including persistentvolumes.
+    roles:
+      - type: ClusterRole  # For getting persistentvolumes
+        rulesType: CUSTOM
+        rules:
+          - apiGroups:
+            - 'rbac.authorization.k8s.io'
+            resources:
+            - 'clusterroles'
+            - 'clusterrolebindings'
+            verbs:
+            - '*'
+          - apiGroups:
+            - '*'
+            resources:
+            - 'persistentvolumes'
```

Comment:
Why do we need permissions for persistent volumes? It doesn't look like we directly create them anywhere in the chart.

Persistent volumes referenced in the `deployment` resource should be auto-created by the K8s API server

Reply:
Oh, you're right, we use here only PV created by the statefulset, let me check make app/install and app/verify without these additional privileges

Reply:
works